router/functionHandler.go

```diff
@@ -154,23 +156,37 @@ func (roundTripper RetryingRoundTripper) RoundTrip(req *http.Request) (resp *htt
 			log.Printf("Calling getServiceForFunction for function: %s", roundTripper.funcHandler.function.Name)
 
 			// send a request to executor to specialize a new pod
-			service, err := roundTripper.funcHandler.executor.GetServiceForFunction(
+			resp, err := roundTripper.funcHandler.executor.GetServiceForFunction(
 				roundTripper.funcHandler.function)
+
 			if err != nil {
-				log.Printf("Err from GetServiceForFunction : %v", err)
+				log.Debugf("Err from GetServiceForFunction : %v", err)
 				// We might want a specific error code or header for fission failures as opposed to
 				// user function bugs.
 				return nil, err
 			}
 
+			if resp.StatusCode != http.StatusOK {
```

Comment:
I understand that the executor server only returns 200 or 4xx or 5xx, but it might be future proof to not reject other 'succesful' status types here. Maybe: `resp.StatusCode >= 400`?